### PR TITLE
Add chapter about PRQL

### PR DESCRIPTION
**Pull request by antaljanosbenjamin**

### CODE REVIEW

**Comment by robot-ch-test-poll4** [comment]
This is an automated comment for commit 4493f652c6136cd0cb255e022c44aacbb1458974 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1300/4493f652c6136cd0cb255e022c44aacbb1458974/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

File: docs/en/tutorial.md [hunk 1]
```diff
@@ -487,6 +487,31 @@ Let's write some queries that join the `taxi_zone_dictionary` with your `trips`
     LIMIT 1000
     ```
 
+## 6. Alternative Query Languages
+
+You can other query languages to query data in ClickHouse using the `dialect` setting. The currently supported dialects are the followings:
```
Comment (DanRoscigno) [inline docs/en/tutorial.md:492]:
```suggestion
You can use other query languages to query data in ClickHouse using the `dialect` setting. The currently supported dialects are:
```

File: docs/en/tutorial.md [hunk 2]
```diff
@@ -487,6 +487,31 @@ Let's write some queries that join the `taxi_zone_dictionary` with your `trips`
...
+Then you can use every PRQL feature that the included PRQL compiler supports:
+
+```prql
+from trips
+aggregate [
+    ct = count
+]
+```
+
+Under the hood ClickHouse will translate the PRQL query into an SQL query and execute it. Once you run all the queries you wanted, you can switch back to the ClickHouse SQL dialect by setting the dialect to `clickhouse`:
```
Comment (DanRoscigno) [inline docs/en/tutorial.md:510]:
```suggestion
Under the hood ClickHouse will translate the PRQL query into an SQL query and execute it. To switch back to the ClickHouse SQL dialect set the dialect to `clickhouse`:
```

**Review by DanRoscigno — APPROVED** [review]
Thanks @antaljanosbenjamin !

**Pull request merged**